features/viewport-segments.yml

```
@@ -0,0 +1,19 @@
+name: Viewport segments
+description: "The `viewport-segment-*` CSS environment variables, the `horizontal-viewport-segments` and `vertical-viewport-segments` media features, and the `viewport.segments` API allow you to adapt your layout to devices where the display is split, such as on foldable devices."
```

Sorry, nitpicky thing here. The [guidelines](https://github.com/web-platform-dx/web-features/blob/main/docs/guidelines.md#descriptions) say to never use "made-up, idiosyncratic, or non-literal example code" and I think the asterisk here is in that category. Maybe something like this would be better?

```suggestion
description: "The CSS environment variables prefixed by `viewport-segment-`, the `horizontal-viewport-segments` and `vertical-viewport-segments` media features, and the `viewport.segments` API allow you to adapt your layout to devices where the display is split, such as on foldable devices."
```